addons/product/models/product_attribute_value.py

```diff
@@ -43,6 +43,9 @@ def _get_default_color(self):
             " to display the color if the attribute type is 'Color'.")
     display_type = fields.Selection(related='attribute_id.display_type')
     color = fields.Integer(string="Color Index", default=_get_default_color)
+    image_1920 = fields.Image(
```

Comment:
The variable name looks strange to me... `attribute_image`? `attribute_color_image`? Something more explicit.